backend/dataall/modules/catalog/api/types.py

```diff
@@ -250,10 +249,11 @@
     ],
 )
 
-GlossaryTermLinkTarget = gql.Union(
+GlossaryTermLinkTarget = gql.ObjectType(
     name='GlossaryTermLinkTarget',
-    type_registry=GlossaryRegistry,
-    resolver=target_union_resolver,
```

Comment:
`target_union_resolver`  import could be removed, the resolver method itself as well + maybe `GlossaryRegistry.find_object_type` as well

Reply:
Thanks @dbalintx - will take this up as part of clean up task in the backlog #741